src/components/Modeler.vue

```diff
@@ -586,6 +588,35 @@ export default {
         }
       });
     },
+
+    getBoundaryEvents(process) {
+      return process.flowElements
+        ? process.flowElements.filter(({$type}) => $type === 'bpmn:BoundaryEvent')
+        : [];
+    },
+    createBoundaryEvent(definition) {
+      let boundaryEvent = this.moddle.create('bpmn:BoundaryEvent', {
+        ...definition,
+        cancelActivity: definition.get('cancelActivity'),
+        attachedToRef: definition.get('attachedToRef'),
+      });
+      boundaryEvent.$parent = definition.$parent;
+      if (definition.get('outgoing').length) {
```

Comment:
This is just my opinion, but I prefer to be more explicit with `length` checks. What are your thoughts on changing this to:
```
definition.get('outgoing').length > 0
```